src/selector.ts

```diff
@@ -393,14 +393,25 @@ class PathElement {
         case 'boolean':
           if (prop[opProp.name] !== (opProp.value.toLowerCase() === 'true')) return undefined;
           break;
-        case 'string':
+        case 'string': {
+          let realString = opProp.value;
+          // look for quoted values, and strip the quotes off if they are there
+          if (opProp.value) {
+            const firstCharacter = realString.charAt(0);
+            const lastCharacter = realString.charAt(realString.length - 1);
+            if ((firstCharacter === '"' && lastCharacter === '"')
+              || (firstCharacter === "'" && lastCharacter === "'")) {
+              realString = realString.substring(1, realString.length - 1);
+            }
```

Comment:
String the matching `'` or `"` characters off the string and set value into realString.  Then use that everywhere else in the string comparison.  Makes it backwards compatible.